lib/identity/realm.js

```diff
@@ -250,6 +251,38 @@ class Realm extends CryptoConsumer {
     return idClient
   }
 
+  async registerFederatedUser(username, data, cryptoKeys, signingKeys) {
+    // eslint-disable-next-line
+    console.log(username, data) // flag if seen in CR
```

Comment:
unsure of what this comment means, or if this log should remain. suggest - more clarity in the comment

Reply:
this is a quick insert I have in VS code, specifically used when something should _not_ go out into the world. Will remove these logs.